catalog/static/config.js

```diff
@@ -13,17 +16,45 @@ if (window.location.hostname === 'quiltdata.com') {
 
     // Quilt auth
     userApi: 'https://stage-auth.quiltdata.com/accounts/api-root',
-    signOutUrl: 'https://stage-auth.quiltdata.com/accounts/logout?next=%2F',
-    /* Dev config
-    team: {
-      name: "TEST"
-    },
-    userApi: 'http://localhost:5002/accounts/api-root',
-    signOutUrl: 'http://localhost:5002/accounts/logout?next=%2F',
-    */
+    // eslint-disable-next-line comma-dangle
+    signOutUrl: 'https://stage-auth.quiltdata.com/accounts/logout?next=%2F'
+
+    // Team feature dev
+    // team: {
+    //   id: "SuperCorp",
+    // },
+    // userApi: 'http://localhost:5002/accounts/api-root',
+    // signOutUrl: 'http://localhost:5002/accounts/logout?next=%2F',
 
     // GitHub
     // userApi: 'https://api.github.com/user',
-    // signOutUrl: '/'
+    // signOutUrl: '/',
   };
 }
+
+const mustHave = {
```

Comment:
All of this really belongs in `constants/config.js`... But I guess it doesn't hurt here.

Reply:
agree for future if we end up needing this logic from another file

Reply:
I meant, we should be checking `config.js.tmpl`, too. Otherwise, it's kind of silly.

Reply:
you're right. coming in clean PR.